frontend/src/reducers/index.js

```diff
@@ -0,0 +1,25 @@
+// index.js file that creates a reducer.
+
+import { CHANGE_MAP_STATE } from "../constants/action-types";
+
+const initialState = {
+  articles: []
+};
+
+function rootReducer(state = initialState, action) {
+  if (action.type === CHANGE_MAP_STATE) {
+    var currState = state.articles;
+    while (currState.length > 0) {
+      currState.pop();
+    }
+    var returnThis = Object.assign({}, state, {
```

Comment:
I suggest that we change the name to something that inform us about what the variable represents. returnThis is a little bit abstract